integration/common_test.go

```diff
@@ -63,12 +68,17 @@ func eksctl(args ...string) *gexec.Session {
 
 func eksctlSuccess(args ...string) *gexec.Session {
 	session := eksctl(args...)
+	return eksctlSuccessWith(params{Args: args})
+}
+
+func eksctlSuccessWith(params params) *gexec.Session {
```

Comment:
I know this is what we are doing right now, but there is fairly major issue with functions like this in ginkgo....

When the assertions fail, you get a trace, but the trace leads to function definition and not the call site. We should start using matchers, and I've started working on that. I about to open a PR that changes `eksctlSuccess` to a matcher (the syntax will be something like `Expect(cmd).To(RunSuccessfully())`). Just FYI.

I'll take what you did here into account make sure it's possible to set custom environment variables.

Reply:
Good to know, and yes, this (among other things) makes it pretty difficult to debug integration tests at the moment.